dev/devicelab/manifest.yaml

```diff
@@ -96,6 +96,13 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["has-android-device"]
 
+  hot_mode_dev_cycle__preview_dart_2_benchmark:
+    description: >
+      Measures the performance of Dart VM hot patching feature under
+      --preview-dart-2 option, that enables Dart 2.0 frontend.
+    stage: devicelab
+    required_agent_capabilities: ["has-android-device"]
+
```

Comment:
see the comment at the top of the file; we should always stage new tests with flaky:true

Reply:
Thanks, good to know. Done.